op_builder/builder.py

```diff
@@ -749,10 +749,17 @@ def nvcc_args(self):
             except ValueError:
                 nvcc_threads = min(os.cpu_count(), 8)
 
-            cuda_major, _ = installed_cuda_version()
+            cuda_major, cuda_minor = installed_cuda_version()
+            if cuda_major > 10:
+                if cuda_major==12 and cuda_minor==5:
```

Comment:
If this is a breaking change going forward, should this be `cuda_minor >= 5`

Reply:
Agreed, good catch

Reply:
@keshavkowshik - this doesn't look to be updated?

Reply:
Now you should see it reflected.

Reply:
Yes thanks!